build/common.sh

```diff
@@ -1427,3 +1437,40 @@ function kube::release::gcs::publish() {
     return 1
   fi
 }
+
+# ---------------------------------------------------------------------------
+# Docker Release
+
+# Releases all docker images to a docker registry specified by KUBE_DOCKER_REGISTRY
+# using tag KUBE_DOCKER_IMAGE_TAG.
+#
+# Globals:
+#   KUBE_DOCKER_REGISTRY
+#   KUBE_DOCKER_IMAGE_TAG
+# Returns:
+#   If new pushing docker images was successful.
+function kube::release::docker::release() {
+  local binaries=(
+    "kube-apiserver"
+    "kube-controller-manager"
+    "kube-scheduler"
+    "kube-proxy"
+  )
+
+  local platforms=(
+    "linux-amd64"
```

Comment:
I agree that consistency is important. But as you can see it's not obvious with what we should be consistent :)

Indeed some images have been pushed with `-arm` suffix. I'll change the naming to just the `arch`.

BTW - I've noticed non-trivial ARM images cannot be build on amd64 machines. That means that we'll not be able to officially release those. How do you envision releasing ARM images?